types/sitemap-generator/sitemap-generator-tests.ts

```diff
@@ -0,0 +1,46 @@
+import SitemapGenerator from 'sitemap-generator';
+
+// $ExpectError
+SitemapGenerator();
+
+SitemapGenerator('https://example.com/');
+
```

Comment:
```suggestion

SitemapGenerator('https://example.com/', {});

```
it supports lazy coder:
https://github.com/lgraubner/sitemap-generator/blob/master/src/index.js#L39